### ISSUE TITLE
Option to force show artist and track name in two vertical strings at widget with small height

### ISSUE THREAD

**Issue by MurzNN** [ismailof/mediacontroller_plus#14]
I integrate `mediacontroller_plus` to my panel, that have 32 px height, but it use space not optimal, here is screenshot:
![image](https://user-images.githubusercontent.com/336662/93429617-1a818700-f8ca-11ea-96ea-8b2a3b21554c.png)

Auto layout is good feature, but will be good to have any manual configuration of layout, at least I want to see artist and track name in two lines, like in bigger layouts, also want to see album picture.

Can you please add option for force 2-line layout of author and song name in small sizes, and force show album picture too? Thanks!

P.S. Also prev/next buttons looks too small in my height, can you increase them little bit?

**Comment by MurzNN** [comment]
Ow, seems this is a duplicate of #12, sorry.

**Comment by ismailof** [comment]
I tried putting two lines on thin panels, but there is really no space to place them without being one over the other. 
From version 0.2.5 I removed the second label, so it's now only one {Artist} - {Track} label. Still not perfect, but should improve the issue a little bit. Let's follow up that issue on #1
 
About the smaller icons, you're right. The shouldn't get so small. Expect them fixed on next version.
